content/collections/guides_and_surveys/en/sdk.md

```diff
@@ -689,82 +567,365 @@ window.electronAPI.startEngagementDebug((data) => {
+    user_id: "user123",
+    device_id: "device456",
+    user_properties: {
+      name: "John Doe",
+      plan: "premium"
+    }
+  }
+});
 ```
 
-Verify that:
-- the `user` object is present
-- `apiKey` is set
-- `stateInitialized` is `true`
-- `decideSuccessful` is `true`
-- `num_guides_surveys` is a non-zero integer if a guide or survey should be display on the page.
+**Format 2: Function that returns EndUser**
+```js
+await window.engagement.boot({
+  user: () => {
+    return {
+      user_id: getCurrentUserId(),
+      device_id: getDeviceId(),
+      user_properties: getUserProperties(),
+      getSessionId: () => getSessionId()
+    };
+  }
+});
+```
 
-### Verify plugin configuration
+**Format 3: Simple string (user ID only)**
+```js
+await window.engagement.boot({
+  user: "user123"
+});
+```
 
-If you use Amplitude Browser SDK 2.0, check the browser's console for errors. If there are none, verify that your code matches code provided in the installation instructions. In particular, ensure that  `amplitude.add(window.engagement.plugin())` is present in the code.
+#### EndUser type
 
-If you see something like `amplitude is not defined` and `cannot read properties of undefined .add()`, this means that the G&S is trying to load before the Amplitude SDK loads. Check your code to ensure that the Amplitude Browser SDK loads before the Guides and Surveys SDK.
+The `EndUser` type includes these properties:
 
-Guides and Surveys requires Browser SDK 2 and doesn't support the legacy Amplitude JavaScript SDK.
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| `user_id` | `string` | User identifier |
+| `device_id` | `string` | Device identifier |
+| `user_properties` | `UserProperties` | Custom user properties |
+| `country` | `string` | Country location data |
+| `region` | `string` | Region location data |
+| `platform` | `string` | Platform identifier |
 
-### Google Tag Manager configuration
+**Required fields:** You must provide at least `user_id` or `device_id`. If neither is provided, the method logs an error and returns early.
 
-If you use Google Tag Manager, ensure you update to the latest Amplitude template.
+#### Integration type
 
-{{partial:admonition type="warning" heading="Google Tag Manager custom tags"}}
-If Guides and Surveys doesn't work with a Google Tag Manager (GTM) custom HTML tag, verify that the **Support document.write** checkbox in the tag configuration is enabled. This setting is required for Guides and Surveys to load properly through GTM.
+The `Integration` type includes a single property currently:
 
-To enable this setting:
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| `track` | `(event: Event) => void` | Optional. Function to track events to your analytics provider |
 
-1. In GTM, navigate to your Amplitude tag.
-2. Expand the **Advanced Settings** section.
-3. Check the **Support document.write** checkbox.
-4. Save and publish your changes.
+#### Boot examples
```

Comment:
This section is a bit awkward since we already have examples above for the parameters which are largely the same

I'd suggest only one examples section, and I'd give mostly examples where we pass `integrations`. Only valid use case after [this PR](https://github.com/amplitude/assistance-browser/pull/806) is shipped for calling without integrations is if you re-boot and want to continue using your existing integrations.